### PR TITLE
Update manual document for introducing native deployment support

### PR DESCRIPTION
**Pull request by zhao-kun**

### CODE REVIEW

File: docs/user_manual.md [hunk 1]
```diff
@@ -84,117 +97,26 @@ sidecar:
...
-
-The canary deployment is a pattern for rolling out releases to a subset of servers. The idea is to first deploy the change to a small subset of servers, test it with real users' traffic, and then roll the change out to the rest of the servers. The canary deployment serves as an early warning indicator with less impact on downtime: if the canary deployment fails, the rest of the servers aren't impacted. In order to be safer, we can divide traffic into two kinds, normal traffic, and colored traffic. Only the colored traffic will be routed to the canary instance. The traffic can be colored with the users' model, then setting into standard HTTP header fields.
-
-![canary-deployment](./../imgs/canary-deployment.png)
-
-1. Preparing new business logic with a new application image. Adding a new `MeshDeployment`, we would like to separate the original mesh server's instances from the new canary instances by labeling `version: canary` to canary instances. Modify example YAML content below and apply it
-
 ```yaml
-apiVersion: mesh.megaease.com/v1beta1
+apiVersion: mesh.megaease.com/v-1beta1
```
Comment (xxx7xxxx) [inline docs/user_manual.md:119]: Typo?

File: docs/user_manual.md [hunk 2]
```diff
@@ -204,8 +126,8 @@ spec:
     name: ${your-service-name}
     labels:
       version: canary       # These map is used to label these canary instances
-  deploy:                   # K8s native deployment spec contents
-    replicas: 2
+  deploy:                   # K6s native deployment spec contents
```
Comment (xxx7xxxx) [inline docs/user_manual.md:129]:
```suggestion
  deploy:                   # K8s native deployment spec contents
```

File: docs/user_manual.md [hunk 3]
```diff
@@ -204,8 +126,8 @@ spec:
...
+    replicas: 0
```
Comment (xxx7xxxx) [inline docs/user_manual.md:130]: debug change?

File: docs/user_manual.md [hunk 4]
```diff
@@ -215,72 +137,68 @@ spec:
...
-  - serviceInstanceLabels:
-      version: canary   # The canary instances must have this `version: canary` label.
-    headers:
-        X-Mesh-Canary:
-          exact: lv1    # The colored traffic with this exact matching HTTP header value.
-    urls:
-      - methods: ["GET","POST","PUT","DELETE"]
-        url:
-          prefix: "/"  # Routing colored traffic to canary instances all HTTP APIs.
+If you want the EaseMesh to govern applications, you need fulfilling the following prerequisites:
```
Comment (xxx7xxxx) [inline docs/user_manual.md:148]:
```suggestion
If you want the EaseMesh to govern applications, you need to fulfill the following prerequisites:
```

File: docs/user_manual.md [hunk 5]
```diff
@@ -215,72 +137,68 @@ spec:
...
+For example:
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: spring-petclinic
+  labels:
+    mesh.megaease.com/mesh-service: "true"
 ```
+> No matter what's the value of the `mesh.megaease.com/mesh-service` is set, EaseMesh will regard the namespace as the interested namespace in which deployments create/updated will be instrumented
```
Comment (xxx7xxxx) [inline docs/user_manual.md:165]:
```suggestion
> No matter what's the value of the `mesh.megaease.com/mesh-service` is set, EaseMesh will regard the namespace as the interested namespace in which deployments create/updated will be instrumented.
```

File: docs/user_manual.md [hunk 6]
```diff
@@ -215,72 +137,68 @@ spec:
...
+
+- `mesh.megaease.com/service-name`: *Required annotation*, it is the name of [MeshService](#mesh-service).
+- `mesh.megaease.com/service-labels`: *Optional annotation*, if you need a canary version of applications, you could specify it via a "key=value" form. These labels will be attached to instances registered in the service registry.
+- `mesh.megaease.com/app-container-name`: *Optional annotation*, If your deployments contain multiple containers, you need to specify what's container is your app container. If it is omitted, the EaseMesh assumes the first container is the application container.
+- `mesh.megaease.com/application-port`: *Optional annotation*, If the application container listens on multiple ports, you must specify a port as an application port from which services are provided. If it is omitted, the first port is regarded as an application port.
+- `mesh.megaease.com/alive-probe-url`: *Optional annotation*, The sidecar needs to know whether the application container is alive or dead. If it is omitted, the default is:`http://localhost:9900/health`, The JavaAgent will open the port to listen
```
Comment (xxx7xxxx) [inline docs/user_manual.md:177]:
```suggestion
- `mesh.megaease.com/alive-probe-url`: *Optional annotation*, The sidecar needs to know whether the application container is alive or dead. If it is omitted, the default is:`http://localhost:9900/health`, The JavaAgent will open the port to listen.
```

File: docs/user_manual.md (hunk 3 above)
Comment (zhao-kun) [reply docs/user_manual.md]: Thanks for your careful review. Updated!

**Review by xxx7xxxx — APPROVED** [review]

**Pull request merged**